zktraffic/base/sniffer.py

```diff
@@ -221,13 +221,14 @@ def handle_packet(self, packet):
       message = self.message_from_packet(packet)
       self.handle_message(message)
     except (BadPacket, StringTooLong, DeserializationError, struct.error) as ex:
+      print("got: %s" % str(ex))
```

Comment:
I think it's only intended to be verbose if the config is set to dump out "bad packets" so mind deleting this?